src/core/content/util.ts

```diff
@@ -73,7 +73,7 @@ export const defaultSeparators = [
  * @param str - Time-string in h:m:s format
  * @returns Seconds
  */
-export function stringToSeconds(str: string | null): number {
+export function stringToSeconds(str?: string | null): number {
```

Comment:
Isn't `str?` flagging the parameter as optional? In what instance would we not pass either string or null here?

Reply:
You’re completely right, I’ll get rid of this.